dockerHashing/src/main/scala/cromwell/docker/registryv2/DockerRegistryV2Abstract.scala

```diff
@@ -131,7 +131,10 @@ abstract class DockerRegistryV2Abstract(override val config: DockerRegistryConfi
   protected def getDockerResponse(token: Option[String], dockerInfoContext: DockerInfoContext)(implicit client: Client[IO]): IO[DockerInfoSuccessResponse] = {
     val requestDockerManifest = manifestRequest(token, dockerInfoContext.dockerImageID, AcceptDockerManifestV2Header)
     lazy val requestOCIManifest = manifestRequest(token, dockerInfoContext.dockerImageID, AcceptOCIIndexV1Header)
-    def tryOCIManifest(err: Throwable) = executeRequest(requestOCIManifest, handleManifestResponse(dockerInfoContext, token))
+    def tryOCIManifest(err: Throwable) = {
+      logger.info("Manifest request failed for docker manifest V2, falling back to OCI manifest", err)
```

Comment:
Seems free, could be useful.
```suggestion
      logger.info(s"Manifest request failed for docker manifest V2, falling back to OCI manifest. Image: ${dockerInfoContext.dockerImageID}", err)
```